README.md

```diff
@@ -259,15 +259,22 @@ $trueOrFalse = $post->saveMeta('foo', 'baz'); // boolean
 
 ### Querying Posts by Custom Fields (Meta)
 
-There are multiples possibilities to query posts by their custom fields (meta). Just use the `hasMeta()` scope under `Post` (actually for all models using the `HasMetaFields` trait) class:
+There are multiples possibilities to query posts by their custom fields (meta) by using scopes on a `Post` (or another other model which uses the `HasMetaFields` trait) class:
+
+To check if a meta key exists, use the `hasMeta()` scope:
+```
+// Finds a published post with a meta flag.
+$post = Post::published()->hasMeta('featured_article'); 
```

Comment:
just add a `->first()` here, please. I would add that, but I'm not on the computer right now 😉💪

Reply:
Done!